pkg/vault/test/foundry/unit/VaultUnit.t.sol

```diff
@@ -79,31 +79,32 @@ contract VaultUnitTest is BaseTest {
         uint256 tokenIndex = 0;
         vault.manualSetAggregateSwapFeeAmount(pool, dai, 0);
 
-        uint256 swapFeeAmountScaled18 = 1e18;
-        uint256 protocolSwapFeePercentage = 10e16;
+        uint256 totalSwapFeeAmountScaled18 = 1.123456789987654321e18;
+        uint256 aggregateSwapFeePercentage = 10.98765432112345e16;
```

Comment:
I believe these values are random? Maybe it'd be good to comment on why these values were chosen.

Reply:
I made it a fuzz test and it passed. Could just do that.

```
    function testComputeAndChargeAggregateSwapFees__Fuzz(uint256 totalSwapFeeAmountScaled18, uint256 aggregateSwapFeePercentage) public {
        totalSwapFeeAmountScaled18 = bound(totalSwapFeeAmountScaled18, 0, MAX_UINT128);
        aggregateSwapFeePercentage = bound(aggregateSwapFeePercentage, 1e12, 50e16);
```